data/scripts/sol-oic-gen.py

```diff
@@ -430,18 +430,18 @@ def generate_object_from_repr_vec_fn_common_c(name, props):
     const struct sol_oic_map_reader *repr_vec, uint32_t decode_mask)
 {
     struct sol_oic_repr_field field;
-    enum sol_oic_map_loop_status end_status;
```

Comment:
We have discussed that we should use status instead of reason. I did those changes some weeks ago to close issue #1721. Have we change that again?